state/store.go

```diff
@@ -633,6 +661,17 @@ func (store dbStore) loadConsensusParamsInfo(height int64) (*cmtstate.ConsensusP
 	return paramsInfo, nil
 }
 
+// saveConsensusTimeoutsInfo saves the timeouts info for the next block to disk.
```

Comment:
[nit] do we know for a fact that the DB writes to disc on every write? I naively assumed the DB would store writes in memory and only write to disc when necessary.

```suggestion
// saveConsensusTimeoutsInfo saves the timeouts info for the next block to DB.
```

Reply:
Thanks!